pendant/patterns/footer.php

```diff
@@ -10,7 +10,7 @@
 <!-- wp:group {"style":{"spacing":{"padding":{"top":"40px","bottom":"40px"}}},"layout":{"inherit":true}} -->
```

Comment:
Another unnecessary block group here.

With block group:

<img width="1200" alt="Screenshot 2024-08-01 at 11 54 19" src="https://github.com/user-attachments/assets/d5a6c514-aceb-4c05-b1a5-0f68a0c633f9">

Without block group (paragraph only with extra padding):

<img width="1191" alt="Screenshot 2024-08-01 at 11 55 59" src="https://github.com/user-attachments/assets/c6db57b6-d09e-414e-bf4b-10c4a8e74a49">

Reply:
I did remove this block group originally, but decided that keeping the group and the empty paragraph makes it easier for people to customise the footer if they did want to revert back to something like how it was before. They just need to click where it says "Type / to choose block" and begin typing

Reply:
(same goes for many of the others)

Reply:
Fair enough 👍